string_search/string_search.java

```diff
@@ -0,0 +1,36 @@
+package string_search;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class string_search {
+
+    public static void main(String[] args) {
+        // System.out.println(args[0]);
+        Scanner scan = new Scanner(System.in);
+        String fileName;
+        System.out.println("Enter file name: ");
+        fileName = scan.next();
+        System.out.println("Enter searchword: ");
+        String search = scan.next();
+        if (!fileName.contains("txt")) {
+            fileName += ".txt";
+        }
```

Comment:
This is not how the program is expected to be run according to instructions, but that probably just being examples of this is probably fine. If you want the program to be done in one call, you could use the args vector

Reply:
We do think this is better :D